nixos/modules/services/networking/charybdis.nix

```diff
@@ -51,46 +51,55 @@ in
         '';
       };
 
+      motd = mkOption {
+        type = types.nullOr types.lines;
+        default = null;
+        description = "Charybdis MOTD.";
```

Comment:
As we're not managing any of the settings in the charybdis-config, we should inform the user here where the motd will be placed so that we can put it in this config.